tests/functional/endpoints.ts

```diff
@@ -42,7 +42,7 @@ describe('Health Endpoint Returns the Correct Fields', async (assert: any) => {
     const response = await getHealth(NODE_PORT)
     const healthData = await response.json()
 
-    const { mongoIsConnected, ipfsInfo, blockchainInfo, networkInfo, walletInfo } = healthData
+    const { mongoIsConnected, ipfsInfo, blockchainInfo, networkInfo, walletInfo, estimatedSmartFeeInfo } = healthData
```

Comment:
You added the `estimatedSmartFeeInfo` but nowhere else I think... is this missing an assertion on it?

Reply:
Yeah I must have accidentally deleted it whenever I merged in master. I'll add it back

Reply:
Ouch :/ Merges can be a PITA